lightning/src/events/bump_transaction.rs

```diff
@@ -147,6 +148,14 @@ pub enum BumpTransactionEvent {
 	/// [`EcdsaChannelSigner::sign_holder_anchor_input`]: crate::sign::ecdsa::EcdsaChannelSigner::sign_holder_anchor_input
 	/// [`build_anchor_input_witness`]: crate::ln::chan_utils::build_anchor_input_witness
 	ChannelClose {
+		/// The `channel_id` of the channel which has been closed.
+		///
+		/// This field will be `None` for objects serialized prior to LDK 0.0.122.
+		channel_id: Option<ChannelId>,
+		/// Counterparty in the closed channel.
+		///
+		/// This field will be `None` for objects serialized prior to LDK 0.0.122.
```

Comment:
I think we can remove this comment on serialization and below since they aren't serialized, no?

https://github.com/lightningdevkit/rust-lightning/blob/a0917e50e88d849302ec2651c4255789ad5c940c/lightning/src/events/mod.rs#L1218-L1227